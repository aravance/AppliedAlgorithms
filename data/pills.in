8 7
BWWWBWBW
10 10
BBBBBBWBBB
4 3
WWWW
12 9
BWWWBWBWWWWW
12 7
BWWWBWBWWWWW
12 12
BWWWBWBWWWWW
20 20
BBBBBBBBBBBBBBBBBBBB
100 100
WWWWWWBWWWWWWWWWBWWWWWWWWWBWWWWWWWWWBWWWWWWWWWBWWWWWWWWWBWWWWWWWWWBWWWWWWWWWBWWWWWWWWWBWWWWWWWWWBWWW
100 50
WWWWWWBWWWWWWWWWBWWWWWWWWWBWWWWWWWWWBWWWWWWWWWBWWWWWWWWWBWWWWWWWWWBWWWWWWWWWBWWWWWWWWWBWWWWWWWWWBWWW
0 0
