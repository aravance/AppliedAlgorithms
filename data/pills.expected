6 7
1 2 3 4 5 6 8 9 10
0
8
0
8 10 11
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
52 57 62 67 72 80 85 89 94 96
0
